playbooks/files/rax-maas/plugins/swift-recon.py

```diff
@@ -66,14 +66,14 @@ def recon_output(for_ring, options=None):
     """
 
     # grab the current release
-    with open("/etc/openstack-release") as f:
+    with open("/etc/rpc-release") as f:
```

Comment:
Maybe we convert this to a loop and search for `rpc-release` and then `osa-release`  which would ensure the check works for both rpc-o and osa.